server/graphql/v2/interface/Account.js

```diff
@@ -322,6 +323,24 @@ const accountFieldsDefinition = () => ({
       return { nodes: result.rows, totalCount: result.count, limit, offset };
     },
   },
+  latestChangelogPublishDate: {
```

Comment:
In my opinion, having this on `Account` does not fit: 99,99% of accounts are not concerned by the changelog. It's a platform feature, so I would add this as a top-level query (in `server/graphql/v2/query/index.js`) rather than a field in Account. @znarf any opinion?

Reply:
@Betree : Yes I was having doubts when I put this on Accounts.js, now it makes sense. I've moved it to the `index.js`. 😄

Reply:
I proposed in another comment to rethink the design and try to better use our generic GraphQL model.